Samples.xcodeproj/project.pbxproj

```diff
@@ -1148,21 +1173,9 @@
 		};
 /* End XCConfigurationList section */
 
-/* Begin XCRemoteSwiftPackageReference section */
-		00C43AEB2947DC350099AE34 /* XCRemoteSwiftPackageReference "arcgis-maps-sdk-swift-toolkit" */ = {
-			isa = XCRemoteSwiftPackageReference;
-			repositoryURL = "https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/";
-			requirement = {
-				kind = upToNextMinorVersion;
-				minimumVersion = 200.0.0-beta;
-			};
-		};
-/* End XCRemoteSwiftPackageReference section */
```

Comment:
It looks like the references to the toolkit got changed. Looks like it may have all happened in 7c1a992d392e2d14c3972bc8fbf48649d9a85e73. I'd look into reverting that commit and re-adding the tag changes only.